datagrepper/app.py

```diff
@@ -59,7 +59,7 @@
 fedmsg.meta.make_processors(**fedmsg_config)
 
 # Initialize a datanommer session.
-dm.init(fedmsg_config["datanommer.sqlalchemy.url"])
+dm.init(app.config.get("DATANOMMER_SQLALCHEMY_URL"))
```

Comment:
I suppose that now it's datanommer's config file, you don't need the `DATANOMMER_` prefix.

Reply:
hmmm, i'd prefer the prefix here, so we know its the datanommer db url (not that a dburl is used in datagrepper, but they are seperate projects technically)

Reply:
Good point.